src/Clowd.WinNative/outline-text/OutlineText.cpp

```diff
@@ -715,9 +715,9 @@ void OutlineText::RenderFontShadow(
 		}
 	}
 
-	pBmpMask->UnlockBits(&bitmapDataMask);
-	pBmpDisplay->UnlockBits(&bitmapDataDest);
-	pBmpFontBodyBackup->UnlockBits(&bitmapDataSrc);
+    if (pBmpMask) pBmpMask->UnlockBits(&bitmapDataMask);
```

Comment:
if any of these 3 locals are null, this method will crash earlier - eg. memory violation on line 652, so we probably don't need to check this here